04_compare_parquets.py

```
@@ -88,10 +97,14 @@ def compare_dataframes(df1, df2, label1, label2):
-                    diff = df1.compare(df2, align_axis=1)
-                    if not diff.empty:
-                        print("\nDetailed comparison (first 5 rows):")
-                        print(diff.head().to_markdown())
+                    # align_axis=1 puts compared columns side by side
+                    try:
+                        diff = df1.compare(df2, align_axis=1)
+                        if not diff.empty:
+                            print("\nDetailed comparison (first 5 rows):")
+                            print(diff.head().to_markdown())
+                    except Exception:
+                        pass
```

Ruff (0.14.11)

`try`-`except`-`pass` detected, consider logging the exception

(S110)